.github/workflows/integrate.yml

```diff
@@ -1,4 +1,4 @@
-name: Integrate
+name: main
```

Comment:
If the name of the workflow changes, it may be worth cascading that change to the file name (i.e. `integrate.yml` -> `main.yml`)